ROADMAP.md

```diff
@@ -69,18 +69,69 @@ There's a loose collection of some BigchainDB 2.0 "TODO" items in the GitHub org
 
 ## Some Goals for BigchainDB 2.1 and Beyond
 
-- Document how to run a production validator so that it's protected from denial-of-service (DoS) attacks and similar issues. See what Tendermint recommends (i.e. "sentry nodes", see issue https://github.com/bigchaindb/bigchaindb/issues/2333).
-- Performance optimization by removing unneeded deepcopy operations.
-- Example projects showing how to use BigchainDB with other decentralized systems, especially Ethereum.
-- Refactoring, especially of the transaction-related code.
+### Short-Term Goals - Low-Hanging Fruits
+
```

Comment:
These are features which are solving a very specific problem. These features while not scoped are expected to be small to medium in implementation complexity.